docs/integrations/sources/rd-station-marketing.md

```diff
@@ -0,0 +1,44 @@
+# RD Station Marketing
+
+RD Station Marketing is the leading Marketing Automation tool in Latin America. It is a software application that helps your company carry out better campaigns, nurture Leads, generate qualified business opportunities and achieve more results. From social media to email, Landing Pages, Pop-ups, even Automations and Analytics.
+
+## Prerequisites
+* An RD Station account
+* A callback URL to receive the first account credential (can be done using localhost)
+* `client_id` and `client_secret` credentials. Access [this link](https://appstore.rdstation.com/en/publisher) to register a new application and start the authentication flow. 
+
+## Airbyte Open Source
+* Start Date
+* client_id
+* client_secret
```

Comment:
Maybe change to Client Id and Client Secret.

Reply:
Done, it really was better!